controllers/kuadrant_controller.go

```diff
@@ -258,16 +258,24 @@ func (r *KuadrantReconciler) registerExternalAuthorizer(ctx context.Context, kOb
 }
 
 func (r *KuadrantReconciler) reconcileSpec(ctx context.Context, kObj *kuadrantv1beta1.Kuadrant) (ctrl.Result, error) {
+	var reconcileAuthorino = true
+
 	if err := r.registerExternalAuthorizer(ctx, kObj); err != nil {
-		return ctrl.Result{}, err
+		if apierrors.IsNotFound(err) {
```

Comment:
The error is not a `Not Found`. OSSM does not deploy IstioOperator CRD, hence the error is:
```
no matches for kind \"IstioOperator\" in version \"install.istio.io/v1alpha1\"
```

I will fix it